### PR TITLE
Kostas/dns checker

### PR DESCRIPTION
**Pull request by xyderos**

### CODE REVIEW

File: DNSChecker/getAuthoritativeServers.py [hunk 1]
```diff
@@ -1,66 +1,42 @@
...
-    default_resolver = dns.resolver.get_default_resolver()
+def getTheIPofAServer(nameOfTheServer):
     
-    name_servers = default_resolver.nameservers[0]
+    temp  = dns.resolver.Resolver().query(nameOfTheServer,'A')
 
-    array_of_domains = domain.split('.')
+    for i in temp.response.answer:
+        for j in i.items:
+            return j.to_text()
```
Comment (KristapsKalnins) [inline DNSChecker/getAuthoritativeServers.py]: You return from the function in a loop ?

File: DNSChecker/getGlueRecords.py [hunk 2]
```diff
@@ -1,76 +1,172 @@
...
 
-    name_servers         = []
+def getTheIPofAServer(nameOfTheServer):
+    
+    temp  = dns.resolver.Resolver().query(nameOfTheServer,'A')
 
-    name                 = dns.name.from_text(domain)
+    for i in temp.response.answer:
+        for j in i.items:
+            return j.to_text()
```
Comment (KristapsKalnins) [inline DNSChecker/getGlueRecords.py]: Same as other file.

**Comment by kasgel** [comment]
I moved the files from DNSChecker to src/checks and tested that they work.
The files inside DNSChecker can be removed.
And as Kristaps mentioned above, the for loop thing can be optimised

**Comment by kasgel** [comment]
Please remove the DNSChecker folder and make the same changes to the files in src/checks
All check files should be located there. I already moved them there but they do not contain your getIPofAServer change.

File: DNSChecker/getGlueRecords.py [hunk 3]
```diff
@@ -169,4 +167,6 @@ def getGlueRecords(domain, list_of_name_servers):
 def run(domain, list_of_name_servers):
     answer = getGlueRecords(domain,list_of_name_servers)
 
-    return {'description':"got glue records", 'result': answer}
\ No newline at end of file
+    return {'description':"got glue records", 'result': answer}
+
+print run("google.com",['ns1.google.com','ns2.google.com'])
```
Comment (KristapsKalnins) [inline DNSChecker/getGlueRecords.py]: Does this print need to be here?

**Review by KristapsKalnins — APPROVED** [review]

**Pull request merged**